src/panel/poi_bloc/poi_bloc_container.js

```diff
@@ -13,7 +13,9 @@ PoiBlocContainer.initBlockComponents = function() {
     try {
       builder = require(`./${poiBlock.panelName}_panel`);
     } catch (err) {
-      const name = poiBlock.panelName.charAt(0).toUpperCase() + poiBlock.panelName.slice(1);
+      const name = poiBlock.panelName.split('_')
```

Comment:
Maybe to avoid this hack we should just put the name in the correct form in the `constants.yml` file :man_shrugging: